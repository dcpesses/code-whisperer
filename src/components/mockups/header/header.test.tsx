import {render} from '@testing-library/react';
import Header from './index';

describe('Header', () => {
  test('Should render as expected', () => {
    const {container} = render(<Header />);
    expect(container).toMatchSnapshot();
  });
});
