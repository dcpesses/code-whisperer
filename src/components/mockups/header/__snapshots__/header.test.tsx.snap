// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Header > Should render as expected 1`] = `
<div>
  <nav
    class="bg-body-tertiary mb-3 py-0 raleway-font navbar navbar-light"
    data-bs-theme="dark"
  >
    <div
      class="container-fluid"
    >
      <span
        class="fw-semibold navbar-brand"
      >
         
      </span>
      <button
        aria-controls="offcanvasNavbar-expand-"
        aria-label="Toggle navigation"
        class="border-0 rounded-0 navbar-toggler collapsed"
        type="button"
      >
        <span
          class="navbar-toggler-icon"
        />
      </button>
      <div
        aria-labelledby="offcanvasNavbarLabel-expand-"
        class="raleway-font offcanvas offcanvas-end"
        id="offcanvasNavbar-expand-"
      >
        <div
          class="offcanvas-header"
        >
          <div
            class="fw-bold offcanvas-title h5"
            id="offcanvasNavbarLabel-expand-"
          >
            Settings
          </div>
          <button
            aria-label="Close"
            class="btn-close"
            type="button"
          />
        </div>
        <div
          class="fw-medium offcanvas-body"
        >
          <div
            class="justify-content-end flex-grow-1 pe-3 navbar-nav"
          >
            <div
              class="nav-item"
            >
              User Info
            </div>
            <span
              class="px-2 navbar-text"
            >
              id: 
            </span>
            <span
              class="px-2 navbar-text"
            >
              access_token: 
            </span>
            <hr
              class="border-bottom my-2"
            />
            <a
              class="nav-link"
              href="#"
              role="button"
              tabindex="0"
            >
              Switch View
            </a>
            <hr
              class="border-bottom my-2"
            />
            <a
              class="nav-link"
              href="#"
              role="button"
              tabindex="0"
            >
              Logout
            </a>
          </div>
        </div>
      </div>
    </div>
  </nav>
</div>
`;
